packages/admin-ui-extensions-react/src/extension-api/index.ts

```diff
@@ -1,7 +1,7 @@
 export {useExtensionApi} from './utils';
-export * from './layout';
-export * from './locale';
-export * from './container';
-export * from './sessionToken';
-export * from './toast';
-export * from './data';
+export * from './LayoutApi';
```

Comment:
Any particular reasons for renaming the files? We don't usually capitalize unless it's a component

Reply:
the docs parser (`dependency-graph.ts`) assumes files are named after their main/default export